save-cloud-charts/save-cloud/templates/s3-deployment.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if not .Values.s3.external -}}
+
+  {{- if eq .Values.profile "dev" -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: s3-secrets
+stringData:
+  s3-storage.endpoint: "http://host.minikube.internal:9000"
```

Comment:
need to add also chart to create s3 storage 
also, I think we will give some name to s3 service and should use it instead of `"http://host.minikube.internal:9000"`

Reply:
Oh ok, will do it

Reply:
Done